apps/marsha/templates/services/app/_deploy_base.yml.j2

```
@@ -81,6 +81,8 @@ spec:
               value: "{{ marsha_postgresql_port }}"
             - name: DJANGO_ALLOWED_HOSTS
               value: "{{ marsha_host | blue_green_hosts }}"
+            - name: DJANGO_CSRF_TRUSTED_ORIGINS
+              value: "{{ ['https://']| product(marsha_host | blue_green_hosts | split(',')) | map('join') | join(',') }}"
```

a bit too smart :sweat_smile:  maybe split in two line by setting the first step in a variable?

---

It looks like Bash. You don't like Bash ?

---

Maybe the following suggestion is less complex to figure out?

```suggestion
              value: "{{ marsha_host | blue_green_hosts | split(',') | map('regex_replace', '^(.*)$', 'https://\\1') | join(',') }}"
```